docs/audits.md

```diff
@@ -14,3 +14,4 @@ Version 2 of the Boson Protocol is an entirely new codebase and architecture fro
 * [View Report - v2.0](audits/Omniscia-Audit-Report-v2.0.pdf)
 * [View Report - v2.1](audits/Omniscia_Audit_Report_Boson_Protocol_V2.1.pdf)
 * [View Report - v2.2](audits/Omniscia_Audit_Report_Boson_Protocol_V2.2.pdf)
+* [View Report - v2.3](audits/Omniscia_Audit_Report_Boson_Protocol_V2.3.pdf)
```

Comment:
```suggestion
* [View Report - v2.3](audits/Omniscia_Audit_Report_Boson_Protocol_v2.3.pdf)
```

Reply:
Alternatively, the pdf file can be renamed to use capital V, the same as the previous two audits.